src/Field/CoordinateReferenceSystemCombo/CoordinateReferenceSystemCombo.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {AutoComplete } from 'antd';
+const Option = AutoComplete.Option;
+
+import './CoordinateReferenceSystemCombo.less';
+
+import { Logger, UrlUtil } from '../../index.js';
+
+/**
+ * Class representating a combo to choose coordinate projection system via a
+ * dropdown menu and / or autocompletion
+ *
+ * @class The CoordinateReferenceSystemCombo
+ * @extends React.Component
+ */
+class CoordinateReferenceSystemCombo extends React.Component {
+
+  /**
+   * The className added to this component.
+   * @type {String}
+   * @private
+   */
+  className = 'react-geo-crs-combo'
+
+  static propTypes = {
+    /**
+     * The className which should be added.
+     * @type {String}
+     */
+    className: PropTypes.string,
+    /**
+     * The API to query for CRS definitions
+     * default: http://epsg.io
+     * @type {String}
+     */
+    epsgIoBaseUrl: PropTypes.string,
```

Comment:
The propertyname could be more general like `crsApiUrl`.

Reply:
changed in 3cc756ffa0